build/distributions/CentOS/7/x86_64/yum_data/packages.yaml

```diff
@@ -6,50 +6,41 @@ chkrootkit:
   :rpm_name: chkrootkit-0.50-4el7.x86_64.rpm
   :source: https://packagecloud.io/simp-project/6_X_Dependencies/packages/el/7/chkrootkit-0.50-4el7.x86_64.rpm
 clamav:
-  :rpm_name: clamav-0.99.2-1.el7.x86_64.rpm
-  :source: http://lug.mtu.edu/epel/7/x86_64/c/clamav-0.99.2-1.el7.x86_64.rpm
+  :rpm_name: clamav-0.100.0-2.el7.x86_64.rpm
```

Comment:
For apr-util-ldap, had to go to vault.centos.org/7.4.1708/os/x86_64/Packages to find the RPM.  Source listed did not work.

Reply:
Fixed CentOS7 link. Tested 6 and the link worked fine.